components/romio/test/ome/io/nio/utests/PixelDataBitTest.java

```diff
@@ -28,46 +28,46 @@ public void setUp()
  
     public void testGetBits()
     {
-    	assertEquals(1.0, data.getPixelValue(0));  // Start byte 1
-    	assertEquals(1.0, data.getPixelValue(1));
-    	assertEquals(1.0, data.getPixelValue(2));
-    	assertEquals(1.0, data.getPixelValue(3));
-    	assertEquals(1.0, data.getPixelValue(4));
-    	assertEquals(1.0, data.getPixelValue(5));
-    	assertEquals(1.0, data.getPixelValue(6));
-    	assertEquals(1.0, data.getPixelValue(7));
-    	assertEquals(1.0, data.getPixelValue(8));  // Start byte 2 
-    	assertEquals(1.0, data.getPixelValue(9));
-    	assertEquals(0.0, data.getPixelValue(10));
-    	assertEquals(0.0, data.getPixelValue(11));
-    	assertEquals(0.0, data.getPixelValue(12));
-    	assertEquals(0.0, data.getPixelValue(13));
-    	assertEquals(0.0, data.getPixelValue(14));
-    	assertEquals(1.0, data.getPixelValue(15));
+        Assert.assertEquals(data.getPixelValue(0), 1.0);  // Start byte 1
+        Assert.assertEquals(data.getPixelValue(1), 1.0);
+        Assert.assertEquals(1.0, data.getPixelValue(2), 1.0);
+        Assert.assertEquals(1.0, data.getPixelValue(3), 1.0);
+        Assert.assertEquals(1.0, data.getPixelValue(4), 1.0);
+        Assert.assertEquals(1.0, data.getPixelValue(5), 1.0);
+        Assert.assertEquals(1.0, data.getPixelValue(6), 1.0);
+        Assert.assertEquals(1.0, data.getPixelValue(7), 1.0);
+        Assert.assertEquals(1.0, data.getPixelValue(8), 1.0);  // Start byte 2 
+        Assert.assertEquals(1.0, data.getPixelValue(9), 1.0);
+        Assert.assertEquals(0.0, data.getPixelValue(10), 0.0);
+        Assert.assertEquals(0.0, data.getPixelValue(11), 0.0);
+        Assert.assertEquals(0.0, data.getPixelValue(12), 0.0);
+        Assert.assertEquals(0.0, data.getPixelValue(13), 0.0);
+        Assert.assertEquals(0.0, data.getPixelValue(14), 0.0);
+        Assert.assertEquals(1.0, data.getPixelValue(15), 1.0);
```

Comment:
Doesn't that mean we sometimes accept a difference of 1.0 and sometimes no difference at all (0.0) (`assertEquals(double actual, double expected, double delta)`), is that intended?